tests/ui/layout/reprc-power-alignment.rs

```diff
@@ -0,0 +1,146 @@
+//@ check-pass
+//@ revisions: aix
+//@[aix] compile-flags: --target powerpc64-ibm-aix
+//@[aix] needs-llvm-components: powerpc
```

Comment:
This test should use our [minicore](https://github.com/rust-lang/rust/blob/cd805f09ffbfa3896c8f50a619de9b67e1d9f3c3/src/doc/rustc-dev-guide/src/tests/minicore.md?plain=1#L9).
```suggestion
//@ compile-flags: --target powerpc64-ibm-aix
//@ needs-llvm-components: powerpc
//@ add-core-stubs
#![feature(no_core)]
#![no_core]
#![no_std]
```